.golangci.yaml

```
@@ -71,6 +71,10 @@ linters:
     - usetesting
     - whitespace
   settings:
+    dupword:
+      # Do not choke on SQL statements like `INSERT INTO things (foo, bar, baz) VALUES (TRUE, TRUE, TRUE)`.
+      # And yes, dupword considers the comma to be part of the word, because... I don't know.
+      ignore: [ "TRUE", "TRUE,", "FALSE", "FALSE,", "NULL", "NULL," ]
```

You should be able to only add the version without `,`:
```suggestion
      ignore: [ "TRUE", "FALSE", "NULL" ]
```

---

Indeed. We can fix that in go-makefile-maker.